internal/debug/flags.go

```diff
@@ -110,17 +116,36 @@ func init() {
 	glogger = log.NewGlogHandler(ostream)
 }
 
+// RotatingFileHandler returns a RotatingFileHandler
```

Comment:
this comment isn't correct, nor helpful.

it also might indicate that the following function could have a better name